templates/solutions/ai/index.html

```diff
@@ -0,0 +1,571 @@
+    <div class="row--50-50 p-section">
+      <div class="col">
+        <h1 class="p-section--shallow">Simplify your enterprise AI journey with trusted open source</h1>
+        <p>Innovate at speed with secure and compliant open source. Get all the AI tooling you need in an integrated stack at a predictable cost.</p>
+        <hr />
+        <a class="p-button--positive" href="/contact-us" aria-controls="ai-modal">Contact Canonical</a>
+      </div>
+      <div class="col">
+        <div class="u-embedded-media">
+          <iframe class="u-embedded-media__element" width="560" height="315" src="https://www.youtube.com/embed/yerW2pOczvA?si=arG1DJW8pqHzmVNU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Why Canonical <br />for enterprise AI?</h2>
+    </div>
+    <div class="col">
+      <ul class="p-list--divided">
+        <li class="p-list__item is-ticked">Run your entire AI/ML lifecycle on a single integrated stack</li>
+        <li class="p-list__item is-ticked">Control your TCO with predictable costs and the best economics in the industry</li>
+        <li class="p-list__item is-ticked">Consume open source AI software at speed, securely</li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>One integrated AI stack</h2>
+    </div>
+    <div class="col">
+      <p>With Canonical, you get a full open source AI solution and one vendor to support it.</p>
+      <p>From GPUs that are optimised for AI use cases to a desktop environment, an integrated MLOps platform, and scalable edge deployments &mdash; do it all on a single stack.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Fast-track compliance <br />with secure AI software</h2>
+    </div>
+    <div class="col">
+      <p>Rapidly access all the open source software you need for AI, and run it securely.</p>
+      <p>We secure and support all of the key open source software in the AI lifecycle so you don't have to. We fix Critical CVE vulnerabilities in under 24h on average, helping you streamline compliance and focus on your AI models.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <h2>Predictable costs and optimal TCO</h2>
+    </div>
+    <div class="col">
+      <p>No software licence fees and predictable pricing per node.</p>
+      <p>Enjoy the freedom of open source, with the best support and economics in the industry.</p>
+      <hr />
```

Comment:
I'm not 100% as this touches the design, but are these supposed to be `hr` (with margin bottom) or `p-rule` without margin above following text?

<img width="1467" alt="image" src="https://github.com/canonical/canonical.com/assets/83575/e5c86238-18a2-4b96-b64a-b1b7be261f09">

@lyubomir-popov ?

The same pattern is used multiple times across this page.